Helper/Quote.php

```diff
@@ -562,7 +568,33 @@ public function isFinancingBySku()
 
         }
 
-        return $quote->getItemsCount() === count($allowed);
+        return (int)$quote->getItemsCount() === count($allowed);
```

Comment:
there is no need to cast items count to integer

Reply:
Really, that's very strange, do you know on what product or product combination?